x-pack/solutions/search/plugins/search_query_rules/public/components/empty_prompt/empty_prompt.tsx

```diff
@@ -218,7 +218,7 @@ export const EmptyPrompt: React.FC<EmptyPromptProps> = ({ getStartedAction }) =>
             <EuiFlexItem grow css={positionRelative}>
               <img
                 src={colorMode === 'DARK' ? queryRulesDarkImg : queryRulesImg}
-                alt="Query Rules"
+                alt="Query rules"
```

Comment:
If we set this, then I think it should be translated. But also I'm not sure this graphic should have an alt-text as it's just a visual graphic.